test/framework/fixtures/kmdFixture.go

```diff
@@ -94,72 +94,72 @@ func (f *KMDFixture) ShutdownImpl(preserveData bool) {
 }
 
 // SetupWithWallet starts kmd and creates a wallet, returning a wallet handle
-func (f *KMDFixture) SetupWithWallet(t TestingT) (handleToken string) {
+func (f *KMDFixture) SetupWithWallet(t TestingTB) (handleToken string) {
 	f.Setup(t)
 	handleToken, _ = f.MakeWalletAndHandleToken()
 	return
 }
 
 // Setup starts kmd with the default config
-func (f *KMDFixture) Setup(t TestingT) {
+func (f *KMDFixture) Setup(t TestingTB) {
 	f.SetupWithConfig(t, "")
 }
 
 // Initialize initializes the dataDir and TestingT for this test but doesn't start kmd
-func (f *KMDFixture) Initialize(t TestingT) {
+func (f *KMDFixture) Initialize(t TestingTB) {
 	f.initialize(f)
-	f.t = t
+	f.t = SynchronizedTest(t)
 	f.dataDir = filepath.Join(f.testDir, t.Name())
 	// Remove any existing tests in this dataDir + recreate
 	err := os.RemoveAll(f.dataDir)
-	require.NoError(t, err)
+	require.NoError(f.t, err)
 	err = os.Mkdir(f.dataDir, 0750)
-	require.NoError(t, err)
+	require.NoError(f.t, err)
 
 	// Set up the kmd data dir within the main datadir
 	f.kmdDir = filepath.Join(f.dataDir, nodecontrol.DefaultKMDDataDir)
 	err = os.Mkdir(f.kmdDir, nodecontrol.DefaultKMDDataDirPerms)
-	require.NoError(t, err)
+	require.NoError(f.t, err)
```

Comment:
Any reason why the require.New pattern is not used in this file?

Reply:
The reason for the `require.New` pattern was to avoid repeating passing the testing.T, since it would require using the `SynchronizedTest` to ensure its synchronized. 
For this fixture, we have that guarantee already, since when we store the KMDFixture.t in Initialize() we always use `SynchronizedTest`.